client/index.html

```diff
@@ -7,16 +7,33 @@
     <script src="lib/angular/angular.js"></script>
     <script src="lib/angular-route/angular-route.js"></script>
     <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.10/angular-ui-router.js"></script>
+    <script src="lib/jquery/dist/jquery.min.js"></script>
```

Comment:
I thought I needed it when I was trying to get the tab highlight working. I forgot to take it out after testing. Will change a push again.